### PR TITLE
Reflects FreeboxOS-java library licensing changes. Switched to EPL 1.0

### PR DESCRIPTION
**Pull request by clinique**
As asked in Issue #1546

Signed-off-by: Gaël L'hopital <glhopital@gmail.com>

### CODE REVIEW

File: addons/binding/org.openhab.binding.freebox/about.html [hunk 1]
```diff
@@ -33,8 +33,7 @@
 		<p><em>
 		<strong>FreeboxOs Java</strong> <br/><br/>
 		<a href="https://github.com/MatMaul/freeboxos-java">MatMaul FreeboxOs Java</a> obtained from Github central repository under 
-		<a href="https://github.com/MatMaul/freeboxos-java/blob/master/LICENSE">GPL version 2.1</a>.<br/>
-		<a href="https://github.com/clinique/freeboxos-java">Clinique fork of FreeboxOs Java - pending merge</a>.
+        <a href="https://github.com/MatMaul/freeboxos-java/blob/master/license-EPL.txt">EPL version 1.0</a>.<br/>
```
Comment (kaikreuzer) [inline addons/binding/org.openhab.binding.freebox/about.html]: could you replace spaces by tabs?

**Comment by clinique** [comment]
Whoops, I did put a big mess in my PR while pushing. Let's restart...

**Comment by clinique** [comment]
We're done !

**Comment by kaikreuzer** [comment]
Thanks!

**Pull request merged**